WalletWasabi/Announcements/ReleaseHighlights.md

```diff
@@ -1,35 +1,40 @@
 ## Release Highlights
-#### 🪲 Bug Fixes<br/>
-#### 💫 Settings Layout & UI Enhancements<br/>
-#### 📈 Tor Update: Upgraded to v13.5.9<br/>
-#### 🚫 Buy Anything Button Disabled<br/>
+#### 🤫 Send to Silent Payments addresses<br/>
+#### 🎁 Bootstrap of our Donation Campaign<br/>
```

Comment:
Donation campaign is not part of the release, I mean, it is a campaign. I would say "Donation Button" 

I would mention `TestNet 4` and the end of `TestNet 3` .